webpack.config.js

```diff
@@ -15,30 +16,12 @@ function generateConfig(name) {
     node: {
       process: false
     },
-    devtool: 'source-map'
+    devtool: 'source-map',
   };
-
-  config.plugins = [
-    new webpack.DefinePlugin({
-      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
-    })
-  ];
-
-  if (uglify) {
-    config.plugins.push(
-      new webpack.optimize.UglifyJsPlugin({
```

Comment:
`mode` is now the canonical way to build with weback.

per the migration guide, the plugins can simply  be removed: https://webpack.js.org/migrate/4/#deprecatedremoved-plugins